core/src/main/scala/coulomb/quantity.scala

```diff
@@ -114,9 +120,15 @@ extension[VL, UL](ql: Quantity[VL, UL])
     transparent inline def /[VR, UR](qr: Quantity[VR, UR])(using div: Div[VL, UL, VR, UR]): Quantity[div.VO, div.UO] =
         div(ql.value, qr.value).withUnit[div.UO]
 
+    transparent inline def tquot[VR, UR](qr: Quantity[VR, UR])(using tq: TQuot[VL, UL, VR, UR]): Quantity[tq.VO, tq.UO] =
+        tq(ql.value, qr.value).withUnit[tq.UO]
+
     transparent inline def pow[P](using pow: Pow[VL, UL, P]): Quantity[pow.VO, pow.UO] =
         pow(ql.value).withUnit[pow.UO]
 
+    transparent inline def tpow[P](using tp: TPow[VL, UL, P]): Quantity[tp.VO, tp.UO] =
+        tp(ql.value).withUnit[tp.UO]
```

Comment:
Consider adding the `infix` modifier?

Reply:
I might.  In scala 3 it's almost as easy to just use backquotes if you really want to go infix
```
q1  `tquot` q2
```

but will consider

Reply:
Yeah, I learned that one today :)